lib/vnet/setup.go

```diff
@@ -188,24 +208,24 @@ func (pm *ProcessManager) Close() {
 	pm.cancel()
 }
 
-// AdminSubcommand is the tsh subcommand that should run as root that will create and setup a TUN device and
-// pass the file descriptor for that device over the unix socket found at socketPath.
+// AdminSetup is supposed to be ran as root. It creates and setups a TUN device and passses the file
```

Comment:
```suggestion
// AdminSetup must run as root. It creates and setups a TUN device and passes the file
```

Assuming the "must" is correct. If it isn't, then what happens if a regular user runs it?